zerver/openapi/zulip.yaml

```
@@ -10052,7 +10052,10 @@ paths:
 
                           The default [message retention policy](/help/message-retention-policy)
                           for this organization.  Pass `"unlimited"` to request that messages
-                          by retained forever (the default).
+                          be retained forever (the default). It can have one special value:
+
+                          * `-1` denoting that the messages will be retained forever for this realm, by default.
+                          Prior to Zulip 3.0, `null` was used for encoding instead of -1.
```

```suggestion
                          Prior to Zulip 3.0, `null` was used for encoding instead of `-1`.
```

Sounds good to me other than that, thanks for the tweak!